languages/javascript/reference/info/immutability.md

```diff
@@ -0,0 +1,6 @@
+## Immutability
+
+This is the concept in which a value cannot be changed after having been created.
+The values are sometimes also referred to as _constant_ or _read-only_ values.
+
+https://developer.mozilla.org/en-US/docs/Glossary/Mutable
```

Comment:
And another question about general styling:

The links in the end, do we want to have them just dangling there, or make up a nice headline for them. Maybe something like

```md
### Further Reading
```

or 

```md
### Sources
```

?